crates/re_viewer/src/viewer_analytics.rs

```diff
@@ -124,17 +143,22 @@ impl ViewerAnalytics {
             let recording_source = match &rec_info.recording_source {
                 RecordingSource::Unknown => "unknown".to_owned(),
                 RecordingSource::PythonSdk(_version) => "python_sdk".to_owned(),
-                RecordingSource::RustSdk => "rust_sdk".to_owned(),
+                RecordingSource::RustSdk(_version) => "rust_sdk".to_owned(),
                 RecordingSource::Other(other) => other.clone(),
             };
 
+            // If we happen to know the Python or Rust version used on the _recording machine_,
+            // then append it to all future events.
+            //
+            // The Python/Rust versions appearing in events always apply to the recording
+            // environment, _not_ the environment in which the viewer is running!
+            if let RecordingSource::RustSdk(version) = &rec_info.recording_source {
+                self.register("rust_version", version.to_string());
+                self.deregister("python_version"); // can't be both!
```

Comment:
maybe we should call these `recording_rust/python_version` for clarity, and the other for `viewer_rust/python_version`?